llvm/test/CodeGen/AMDGPU/lround.ll

```diff
@@ -0,0 +1,1036 @@
+; NOTE: Assertions have been autogenerated by utils/update_llc_test_checks.py UTC_ARGS: --extra_scrub --version 5
+; RUN: llc -global-isel=0 -mtriple=amdgcn -mcpu=gfx900 < %s | FileCheck -check-prefixes=GFX9-SDAG %s
+; RUN: llc -global-isel=1 -mtriple=amdgcn -mcpu=gfx900 < %s | FileCheck -check-prefixes=GFX9-GISEL %s
```

Comment:
Should add a common GFXN check prefix for all of these

Reply:
I have tried adding it and there are no common lines